cmake/embed_clang.cmake

```diff
@@ -0,0 +1,140 @@
+if(EMBED_CLANG)
+  include(ExternalProject)
+  include(ProcessorCount)
+
+  if(CMAKE_BUILD_TYPE STREQUAL "Debug")
+    set(EMBEDDED_BUILD_TYPE "RelWithDebInfo")
+  elseif(CMAKE_BUILD_TYPE STREQUAL "Release")
+    set(EMBEDDED_BUILD_TYPE "MinSizeRel")
+  else()
+    set(EMBEDDED_BUILD_TYPE ${CMAKE_BUILD_TYPE})
+  endif()
+
+  if(${LLVM_VERSION} VERSION_EQUAL "9" OR ${LLVM_VERSION} VERSION_GREATER "9" )
```

Comment:
Any reason not to do `VERSION_GREATER_EQUAL` rather than `VERSION_EQUAL OR VERSION_GREATER`?

Reply:
Didn't know about that operator in cmake, i'm a little rusty - thanks I'll change it!